Formula/nonpareil.rb

```diff
@@ -0,0 +1,34 @@
+class Nonpareil < Formula
+  # cite Rodriguez_2014: "https://doi.org/10.1093/bioinformatics/btt584"
+  desc "Estimates coverage in metagenomic datasets"
+  homepage "http://enve-omics.ce.gatech.edu/nonpareil"
+
+  url "https://github.com/lmrodriguezr/nonpareil/archive/v3.3.1.tar.gz"
+  sha256 "6fc9caf915f80704876fcd7b0a1fb3beda664fdb840fa0866bdfa1cb76df1a34"
+  head "https://github.com/lmrodriguezr/nonpareil.git"
+
+  depends_on "r"
+  depends_on "open-mpi" => :optional
+
+  def install
+    r_library = lib/"R"/r_major_minor
+    r_library.mkpath
+    inreplace "Makefile", "CMD INSTALL", "CMD INSTALL --library=#{r_library}"
+    system "make", "nonpareil"
+    system "make", "mpicpp=#{ENV["MPICXX"]}", "nonpareil-mpi" if build.with? :"open-mpi"
```

Comment:
`brew` clears the environment to ensure that the build is reproducible, so `ENV["MPICXX"]` will not be defined. I'd suggest…
```ruby
system "make", "nonpareil-mpi" if build.with? "open-mpi"
```
Have you tested that this option works?